native_client/deepspeech.cc

```diff
@@ -56,13 +59,36 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
     return;
   }
 
-  Status status = NewSession(SessionOptions(), &mPriv->session);
+  Status status;
+  SessionOptions options;
+  bool is_mmap = std::string(aModelPath).find(".pbmm") != std::string::npos;
```

Comment:
Simple way to easily distinguish between both. Trying to use the wrong file will fail, anyway.